sky/cli.py

```diff
@@ -3179,8 +3180,11 @@ def _output():
                     yield from k8s_realtime_table.get_string()
                     k8s_node_table = _get_kubernetes_node_info_table(context)
                     yield '\n\n'
+                    # TODO(Doyoung): Update the message with the multi-host TPU
+                    # support.
                     yield (f'{colorama.Fore.CYAN}{colorama.Style.BRIGHT}'
-                           f'Kubernetes per node GPU availability'
+                           f'Kubernetes per node accelerator availability '
+                           '(Note: Multi-host TPUs are not supported.)'
```

Comment:
Can we have this Note show up only when multi-host TPU nodes are detected in the cluster?

Reply:
Yes, updated at [c1ee117](https://github.com/skypilot-org/skypilot/pull/3947/commits/c1ee117ebef292fca0e775415955e41437497eb5).

```
$ kubectl get nodes
NAME                                               STATUS   ROLES    AGE    VERSION
gke-mix-tpu-test-txia-default-pool-669a5bd9-gd5s   Ready    <none>   129m   v1.30.5-gke.1014001
gke-mix-tpu-test-txia-default-pool-669a5bd9-wxbx   Ready    <none>   129m   v1.30.5-gke.1014001
gke-tpu-430ee63e-cbs3                              Ready    <none>   120m   v1.30.5-gke.1014001
gke-tpu-430ee63e-khrj                              Ready    <none>   120m   v1.30.5-gke.1014001
gke-tpu-65979ccc-41hz                              Ready    <none>   125m   v1.30.5-gke.1014001
gke-tpu-93e8af66-fdxd                              Ready    <none>   127m   v1.30.5-gke.1014001

$ sky show-gpus --cloud kubernetes
Kubernetes GPUs (context: gke_skypilot-375900_us-south1-a_mix-tpu-test-txia)
GPU                   QTY_PER_NODE  TOTAL_GPUS  TOTAL_FREE_GPUS
tpu-v5-lite-podslice  1, 2, 3, 4    5           5

Kubernetes per node accelerator availability (Note: Multi-host TPUs are not supported.)
NODE_NAME                                         GPU_NAME              TOTAL_GPUS  FREE_GPUS
gke-mix-tpu-test-txia-default-pool-669a5bd9-gd5s  None                  0           0
gke-mix-tpu-test-txia-default-pool-669a5bd9-wxbx  None                  0           0
gke-tpu-65979ccc-41hz                             tpu-v5-lite-podslice  4           4
gke-tpu-93e8af66-fdxd                             tpu-v5-lite-podslice  1           1

$ kubectl get nodes
NAME                                               STATUS   ROLES    AGE    VERSION
gke-mix-tpu-test-txia-default-pool-669a5bd9-gd5s   Ready    <none>   162m   v1.30.5-gke.1014001
gke-mix-tpu-test-txia-default-pool-669a5bd9-wxbx   Ready    <none>   162m   v1.30.5-gke.1014001
gke-tpu-65979ccc-41hz                              Ready    <none>   159m   v1.30.5-gke.1014001
gke-tpu-93e8af66-fdxd                              Ready    <none>   161m   v1.30.5-gke.1014001

$ sky show-gpus --cloud kubernetes
Kubernetes GPUs (context: gke_skypilot-375900_us-south1-a_mix-tpu-test-txia)
GPU                   QTY_PER_NODE  TOTAL_GPUS  TOTAL_FREE_GPUS
tpu-v5-lite-podslice  1, 2, 3, 4    5           5

Kubernetes per node accelerator availability
NODE_NAME                                         GPU_NAME              TOTAL_GPUS  FREE_GPUS
gke-mix-tpu-test-txia-default-pool-669a5bd9-gd5s  None                  0           0
gke-mix-tpu-test-txia-default-pool-669a5bd9-wxbx  None                  0           0
gke-tpu-65979ccc-41hz                             tpu-v5-lite-podslice  4           4
gke-tpu-93e8af66-fdxd                             tpu-v5-lite-podslice  1           1
```

Reply:
nit: maybe change to sth like `detected nodes xxx using multi host TPU, which is not supported now. skip showing them.`.